btree/btree.c

```diff
@@ -2970,21 +2987,21 @@ static void level_cleanup(struct m0_btree_oimpl *oi, struct m0_be_tx *tx)
 }
 
 /**
- * Adds unique node descriptor address to m0_btree_oimpl::i_callback.
+ * Adds unique node descriptor address to m0_btree_oimpl::i_capture structure.
  */
-static void btree_callback_add(void **arr, void *addr)
+static void btree_callback_add(struct node_capture *arr, void *addr)
```

Comment:
Take struct m0_btree_oimpl * as first parameter instead of struct node_capture * as the 1st parameter.
Also the 2nd parameter sho;uld be struct nd * and not void *.
Also 3rd parameter record index should be passed too.